setup.py

```diff
@@ -257,7 +257,7 @@ def setup_package():
             'Development Status :: 4 - Beta',
             'Intended Audience :: Science/Research',
             'Topic :: Scientific/Engineering',
-            'License :: OSI Approved :: GNU Library or Lesser General Public ' +
+            'License :: OSI Approved :: GNU Library or Lesser General Public '
```

Comment:
Why the change here? Was the license not rendering as expected (e.g. with the addition of the 'License (LGPL)' line below?

Reply:
If I understand what you are doing here the plus just isn't needed, eg:
```python
a = 'version ' + '5.7'
b = 'version ' '5.7'
assert a == b
```
If you feel that it will make it more clear I can add it back, but it does knock the line length under 80.

Reply:
Ah, right you are - I was just following examples a while back I think that used that syntax.